### PR TITLE
fix(DB/skinning_loot_template): Create new skinning entries for different mobs in TBC

### PR DESCRIPTION
**Pull request by Pyromanihack**
## Changes Proposed:
-  Create a new skinning loot entry with 80% chance (Knothide Leather) and 20% chance (Knothide Leather Scraps)
-  Create a new skinning loot entry with 100% chance (Knothide Leather)
-  Updated different mobs (see Source) with the right skinning loot percentage

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/14486

## SOURCE:

https://github.com/azerothcore/azerothcore-wotlk/issues/14486

## Tests Performed:

- I checked the updated values in the DB

## How to Test the Changes:

1. learn Skinning
2. `.additem 7005` (skinning knife)
3. `.go c 74891`
4. Skin the creature

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Review by Gultask — CHANGES_REQUESTED** [review]

**Comment by Gultask** [comment]
https://www.azerothcore.org/wiki/loot_template explains why there are chances with the value 0.
Essentially not having it grouped and having 2 items being like:

- 80% chance
- 20% chance

can make:

1. Both drop at 16% chance (0.8*0.2)
2. Neither drop at 16% chance

While grouping it will **ensure** one of either drops.

**Comment by Pyromanihack** [comment]
So basically when I have to set a skinning loot, it's better to put the loot in a group (considering that a creature drop at least 1 skin)?

**Comment by Gultask** [comment]
Yep. Unless it's only 1 item with 100% chance.
But most entries look like this:
image

GroupID also handles 0-chance (or equal chance) items.
In the above case, if neither the Leather or Fel Hide drops, then the scraps will be guaranteed, so it's basically at 70% chance.

File: data/sql/updates/pending_db_world/skinning.sql [hunk 1]
```diff
@@ -0,0 +1,18 @@
+-- Create a new skinning loot with right percentage
+DELETE FROM `skinning_loot_template` WHERE (`Entry` = 70068) AND (`Item` IN (21887, 25649, 35229));
+INSERT INTO `skinning_loot_template` (`Entry`, `Item`, `Reference`, `Chance`, `QuestRequired`, `LootMode`, `GroupId`, `MinCount`, `MaxCount`, `Comment`) VALUES
+(70068, 21887, 0, 80, 0, 1, 1, 1, 1, 'Knothide Leather'),
+(70068, 25649, 0, 0, 0, 1, 1, 1, 3, 'Knothide Leather Scraps'),
```
Comment (Gultask) [inline data/sql/updates/pending_db_world/skinning.sql:5]:
```suggestion
INSERT INTO `skinning_loot_template` (`Entry`, `Item`, `Reference`, `Chance`, `QuestRequired`, `LootMode`, `GroupId`, `MinCount`, `MaxCount`, `Comment`) VALUES
(70068, 21887, 0, 80, 0, 1, 1, 1, 1, 'Knothide Leather'),
(70068, 25649, 0, 0, 0, 1, 1, 2, 3, 'Knothide Leather Scraps'),
```
Sources say mincount should be 2. Just one last observation :)

**Review by Gultask — APPROVED** [review]

**Comment by BeefSteaker** [comment]
Tested, works as expected

**Pull request merged**